syncer/syncer.go

```diff
@@ -1341,14 +1341,20 @@ func (s *Syncer) syncDML(
 	defer ticker.Stop()
 	for {
 		// resets the time interval for each loop to prevent a certain amount of time being spent on the previous ticker
-		// execution to `executeSQLs` resulting in the next tikcer not waiting for the full waitTime.
+		// execution to `executeSQLs` resulting in the next ticker not waiting for the full waitTime.
```

Comment:
It seems ticker is reset every time, why not use `time.Timer`?

Reply:
https://github.com/pingcap/dm/pull/1939/commits/4bdb3cc848c92eae5f58009dd204aabfa4de5c99